### PR TITLE
Update SAML tests for changes due to RFC 9110

### PR DESCRIPTION
**Pull request by c00crane**
Update expected behavior for tests using embedded spaces in the saml header name.

### CODE REVIEW

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.saml.sso_fat.jaxrs.config

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.saml.sso_fat.jaxrs.config
#libby

File: dev/com.ibm.ws.security.saml.sso_fat.jaxrs.config/fat/src/com/ibm/ws/security/saml/fat/jaxrs/config/IDPInitiated/RSSamlIDPInitiatedMiscConfigTests.java [hunk 1]
```diff
@@ -155,24 +155,81 @@ public void RSSamlIDPInitiatedConfigTests_headerName_nonEmpty() throws Exception
...
 
+    /**
+     * Pass header as Authorization=<header>=<value>
+     *
+     * @throws Exception
+     */
+    @Test
+    public void RSSamlIDPInitiatedConfigTests_headerName_nonEmptyWithSpaces_formatHeaderEqualsValue() throws Exception {
+
+        common_headerName_withSpaces(SAMLConstants.SAML_HEADER_1, "Some value with  spaces.", SAMLConstants.OK_STATUS);
```
Comment (ayoho) [inline dev/com.ibm.ws.security.saml.sso_fat.jaxrs.config/fat/src/com/ibm/ws/security/saml/fat/jaxrs/config/IDPInitiated/RSSamlIDPInitiatedMiscConfigTests.java]: The always present naming conundrum. For the constants for the header format types, you did effectively just name what the variables are in the method names. So what if we did something like:
```
SAMLConstants.SAML_HEADER_1 -> SAMLConstants.HEADER_FORMAT_AUTHZ_NAME_EQUALS_VALUE
SAMLConstants.SAML_HEADER_2 -> SAMLConstants.HEADER_FORMAT_AUTHZ_NAME_EQUALS_QUOTED_VALUE
SAMLConstants.SAML_HEADER_3 -> SAMLConstants.HEADER_FORMAT_AUTHZ_NAME_SPACE_VALUE
SAMLConstants.SAML_HEADER_4 -> SAMLConstants.HEADER_FORMAT_NAME_EQUALS_VALUE
```

**Review by ayoho — DISMISSED** [review]

File: dev/com.ibm.ws.security.saml.sso_fat.jaxrs.config/fat/src/com/ibm/ws/security/saml/fat/jaxrs/config/IDPInitiated/RSSamlIDPInitiatedMiscConfigTests.java (hunk 1 above)
Comment (c00crane) [reply dev/com.ibm.ws.security.saml.sso_fat.jaxrs.config/fat/src/com/ibm/ws/security/saml/fat/jaxrs/config/IDPInitiated/RSSamlIDPInitiatedMiscConfigTests.java]: Let me see how many files that'll affect - Those constants have been around for a while... there are 5t/5f and 6t/6f too

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.saml.sso_fat.jaxrs.config
#libby

**Review by ayoho — APPROVED** [review]

**Pull request merged**